src/ds3.h

```diff
@@ -18,7 +18,6 @@
 
 #include <stdint.h>
 #include <string.h>
-#include <curl/curl.h>
```

Comment:
curl.h was included in commit https://github.com/SpectraLogic/ds3_c_sdk/commit/7662c26182be0519b07490c6bb4768691b9d6a33 so that the C SDK could expose its own macros and not rely on a user having to directly use lib curl's macros.  Since CURL_READFUNC_ABORT is still referenced in ds3.h, I recommend leaving the include in.